packages/perseus/src/renderer.tsx

```diff
@@ -249,9 +245,6 @@ class Renderer
 
     _widgetContainers: Map<string, WidgetContainer> = new Map();
 
-    lastRenderedMarkdown: React.ReactNode;
-    // @ts-expect-error - TS2564 - Property 'reuseMarkdown' has no initializer and is not definitely assigned in the constructor.
-    reuseMarkdown: boolean;
```

Comment:
So this was basically cashing the result of the render for reuse. Super weird. Maybe we want to separate out parsing/rendering and that's what this was for? To prevent reparsing Markdown? But if so this isn't the way we should do it and I'm not sure I agree that we should do it.